File: php/src/Test/MyClassTest.php
<?php

namespace Foo\Test;

use Foo\MyClass;

class MyClassTest extends \PHPUnit_Framework_TestCase
{

    public function testMyMethodAllwaysReturnsTrue()
    {
        $myClass = new MyClass();
        $this->assertTrue($myClass->myMethod());
    }

}

File: ruby/test/my_class_test.rb

$:.unshift File.join(File.dirname(__FILE__),'..','lib')

require_relative '../src/my_class'
require 'test/unit'

class MyClassTest < Test::Unit::TestCase

  def test_foo
    @myClass = MyClass.new
    assert_equal(true, @myClass.my_method)
  end
end
